### PR TITLE
Refactor of kitchen-vcair plugin, initial public release

### PR DESCRIPTION
**Pull request by adamleff**
Refactor of vulk's work, added tests, added rubocop, support for vCloud OnDemand

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+To apply the Apache License to your work, attach the following
+boilerplate notice, with the fields enclosed by brackets "{}"
+replaced with your own identifying information. (Don't include
+the brackets!)  The text should be enclosed in the appropriate
+comment syntax for the file format. We also recommend that a
+file or class name and description of purpose be included on the
+same "printed page" as the copyright notice for easier
+identification within third-party archives.
+
+Copyright {yyyy} {name of copyright owner}
```
Comment (jjasghar) [inline LICENSE.txt]: Forgot to fill this in.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,188 @@
...
+All of the above settings can be set globally (in the top-level `driver` section), or can be set individually for each platform. For example, you may wish to set your vDC and network globally, but set your catalog and image for each individual platform, and increase the vCPUs/RAM assigned to your windows node:
+
+```yaml
+driver:
+  name: vcair
+  vcair_username: user@domain.com
+  vcair_password: MyS33kretPassword
+  vcair_api_host: some-host.vchs.vmware.com
+  vcair_org: M12345678-4321
+  vdc_name: MyCompany VDC 1
```
Comment (jjasghar) [inline README.md]: Are spaces ok here? You may want to mention that somewhere.

File: LICENSE.txt (hunk 1 above)
Comment (adamleff) [inline LICENSE.txt]: Not meant to be filled in.  That's a template to be used per the paragraph right above it.

File: README.md (hunk 2 above)
Comment (adamleff) [inline README.md]: It depends on how the VDC was created.  We just pass what the user supplies to the API.  If the VDC has spaces in the name, you HAVE to include the spaces here.  Do you think that requires documenting on our side when it's a construct imposed by VMware, not us?

File: kitchen-vcair.gemspec [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  spec.summary       = 'A Test Kitchen vCloud Air driver built on Fog'
+  spec.homepage      = 'https://github.com/chef-partners/kitchen-vcair'
+  spec.license       = 'Apache 2.0'
+
+  spec.files         = `git ls-files -z`.split("\x0")
+  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
+  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
+  spec.require_paths = %w(lib)
+
+  spec.required_ruby_version = '>= 1.9.3'
```
Comment (jjasghar) [inline kitchen-vcair.gemspec]: Surprised you didn't force 2+ ruby

Comment (adamleff) [inline kitchen-vcair.gemspec]: I can change that.  I left what Vulk specified in their initial offering.

**Comment by jjasghar** [comment]
It looks good apart from what i asked above. I do suggest you spin up travis also with this, at least to gate unit tests.

Other than that :+1:

**Pull request merged**